samples/CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ target_link_libraries(null vfio-user-static pthread)
 LINK_DIRECTORIES(${CMAKE_BINARY_DIR}/lib)
 add_executable(gpio-pci-idio-16 gpio-pci-idio-16.c)
 target_link_libraries(gpio-pci-idio-16 vfio-user)
```

Comment:
I think here it should be linking to vfio-user-shared?

Reply:
yeah, that's much better ,thanks.